code/addons/test/src/vitest-plugin/index.ts

```diff
@@ -36,7 +38,7 @@ const extractTagsFromPreview = async (configDir: string) => {
   return previewConfig.getFieldValue(['tags']) ?? [];
 };
 
-export const storybookTest = (options?: UserOptions): Plugin => {
+export const storybookTest = async (options?: UserOptions): Promise<Plugin> => {
```

Comment:
Made async, so we can call `loadAllPresets` and re-use the same instance in all hooks